bindata/network/ovn-kubernetes/managed/alert-rules-control-plane.yaml

```diff
@@ -15,6 +15,8 @@ spec:
     rules:
     - record: cluster:ovnkube_controller_egress_routing_via_host:max
       expr: max(ovnkube_controller_egress_routing_via_host)
+    - record: cluster:ovnkube_controller_admin_network_policies_db_objects:max
+      expr: max(ovnkube_controller_admin_network_policies_db_objects)
```

Comment:
maybe this needs to have some kind of label element to it?

Reply:
i think this is max across all time-series so if youre happy with that, you should be good.

Whats your goal here?  If your goal is to see the max ever ANP pol objs over all tables and over the life time of the cluster, then youre good i think with this expr. 

It might be more interesting to see a max over time periods if you want to gauge the usage of ANP over time periods instead of over time then. Where ever this is being exported to, you can then simply do a max() again then if you wanted to ask the question "whats the max seen over the lifetime of the cluster" but youve also the option of looking over time periods.

Reply:
so my goal here is to see max(acls) and max(address-sets)
my confusion is stemming from the fact that this metric has labels that are table names and how can I see the total number of objects per label?

Reply:
Don't you want to preserve the table_name label? If yes, you need  `max by(table_name) (ovnkube_controller_admin_network_policies_db_objects)`.

Reply:
thanks @simonpasquier ! yes I want to preserve the table_name, correct. Let me update the alert expression based on that

Reply:
I have repushed now also adding both PRs we had: https://github.com/ovn-org/ovn-kubernetes/pull/4248#issuecomment-2110930821 and https://github.com/ovn-org/ovn-kubernetes/pull/4254#issuecomment-2059847242 to the mix